pallets/liquidity-pools-gateway/queue/src/benchmarking.rs

```diff
@@ -28,6 +28,7 @@ mod benchmarks {
 	#[benchmark]
 	fn process_message() -> Result<(), BenchmarkError> {
 		let caller: T::AccountId = account("acc_0", 0, 0);
+		//TODO(cdamian): Add a way of retrieving the "heaviest" message.
```

Comment:
Reminder. But I would leave it as it is in this PR. I need to know how to fit this with the batch solution. It will need another thought

Reply:
BTW, being defensive means that we can avoid these benchmarks I think. So there is no longer an issue here. We can even remove the benchmarks & weights from this 🤔

Reply:
I'm OK with this, @wischli?